docs/docs/start-here.md

```diff
@@ -38,53 +42,58 @@ questions, which you can look for in the [Nullness Design FAQ]. If you like,
 
 ### Begin with one of
 
-* The current [User Guide] draft
-* The [Javadoc], which is not a *tutorial* walkthrough, but is thorough,
-  specific, and current.
+*   The [User Guide].
+*   The [Javadoc], which is not a *tutorial* walkthrough, but is thorough,
+    specific, and current.
```

Comment:
I think that the word "current" here was previous a contrast with the User Guide. Now that the User Guide is also current, I'd remove it.

(The spec is of course still a little out of date, but you cover that below, and it's of interest to fewer people.)